app/services/ping.rb

```diff
@@ -0,0 +1,65 @@
+class Ping
+  class << self
+    def ping
+      {
+        'foreman': {
+          database: ping_database
+        }
+      }.merge(plugins(:ping))
+    end
+
+    def statuses
+      {
+        'foreman': {
+          version: SETTINGS[:version].full,
+          api: {
+            version: 2
+          },
+          plugins: Foreman::Plugin.all,
+          smart_proxies: statuses_smart_proxies,
+          compute_resources: statuses_compute_resources
+        }
+      }.merge(plugins(:status))
```

Comment:
I wonder if it makes sense to also include ping results into statuses output, so if you have an account you can monitor uptime with just a single call.

Reply:
If there are no rejections from anyone, I'd add: `...}.merge(plugins(:status)).merge(ping)`